nav2_tasks/include/nav2_tasks/is_stuck_condition.hpp

```diff
@@ -199,11 +188,12 @@ class IsStuckCondition : public BT::ConditionNode, public rclcpp::Node
         curr_odom.twist.twist.linear.x - prev_odom.twist.twist.linear.x);
       current_accel_ = vel_diff / dt;
 
-      if (current_accel_ < minimum_measured_accel_) {
-        minimum_measured_accel_ = current_accel_;
-        RCLCPP_DEBUG(get_logger(),
-          "Minimum accel detected, dt: %.6f s, vel diff: %.6f m/s, accel: %.6f m/s^2",
-          dt, vel_diff, current_accel_);
+      if (vel_diff > (odom_error_ / 2.0)) {
```

Comment:
I don't like magic numbers... can you make a function that tales in `vel_diff` called `IsZero()` or similar that looks at this value and determines if close enough to zero? it would make cleaner the intent

Reply:
@SteveMacenski you're looking at an outdated version, the new one doesn't have that check.

Reply:
Oh weird